includes/ets_admin_qa_function.php

```diff
@@ -451,19 +492,22 @@ public function admin_qa_save( $productId ){
 				if ( !empty(trim($value['answer'])) && !empty(trim($value['user_email'])) && (trim($value['answer']) != trim($before_save[$key]['answer']) && !empty( trim( $before_save[$key]['answer'] )  ) ) )
 				{
 
-			 		$subject = __("Answer to Your Question was Updated",'ets_q_n_a'). ': ' . get_bloginfo('name');
+			 		$subject =apply_filters("wc_qa_answer_updated_mail_subject" ,__("Answer to Your Question was Updated",'ets_q_n_a'). ': ' . get_bloginfo('name'));
 			 		$message = "Dear " . $userName . ",<br><br>";
 			 		$message .= "<a href='$site_url'>" . $site_name . "</a> updated an answer to your question on the product <a href='$url'> " . $productTitle ."</a>:  <br><div style='background-color: #FFF8DC;border-left: 2px solid #ffeb8e;padding: 10px;margin-top:10px;'>". $answers ."</div>";
 
+			 		$message = apply_filters("wc_qa_answer_updated_mail_message", $message, $productTitle, $answers);
+
 				    $res = wp_mail($to, $subject, $message);
 				 
 				// First time answer    
 				} elseif ( empty( trim( $before_save[$key]['answer'] ) ) && !empty( trim( $value['answer'] ) )  && !empty(trim($value['user_email'])) ) {  
 
-			 		$subject = __("Your Question was Answered",'ets_q_n_a'). ': ' . get_bloginfo('name');
+			 		$subject = apply_filters("wc_qa_new_answer_mail_subject", $subject);
```

Comment:
How we will get `$subject` variable here?

Reply:
i had added but i thing its remove accidentally  i have added again